src/App.js

```diff
@@ -23,21 +24,37 @@ const dummyMovie =  {id: 1,
 
 function App() {
 
-  const [movies, setMovies] = useState(movieData.movies)
+  const [movies, setMovies] = useState([])
   const [singleMovie, setSingleMovie] = useState(null)
```

Comment:
Nice work setting the single movie state here! Just a suggestion: it’s recommended to assign a specific data type to the initial state. For example, if the single movie is an object, the initial state should be an empty object instead of null. Assigning null to a state can obscure the state’s type and make debugging more challenging.